alpha/declcfg/model_to_declcfg.go

```diff
@@ -115,7 +115,7 @@ func traverseModelChannels(mpkg model.Package) ([]Channel, []Bundle) {
 	return channels, bundles
 }
 
-func modelRelatedImagesToRelatedImages(relatedImages []model.RelatedImage) []RelatedImage {
+func ModelRelatedImagesToRelatedImages(relatedImages []model.RelatedImage) []RelatedImage {
```

Comment:
Does this need to be exported?  All references I could find were local, and we'd like to minimize exports due to Hyrum's Law.